diff --git a/src/configure b/src/configure
index 0948f4a..da867ea 100755
--- a/src/configure
+++ b/src/configure
@@ -2299,7 +2299,7 @@ else
     prefix="${ac_default_prefix}"
   fi
   libpltdir="${libdir}/racket"
-  collectsdir="${libdir}/racket/collects"
+  collectsdir="${datadir}/racket/collects"
   includepltdir="${includedir}/racket"
   docdir="${datadir}/racket/doc"
   MAKE_COPYTREE=copytree
